src/components/SecondaryPanes/Frames/Group.js

```diff
@@ -14,12 +14,11 @@ import "./Group.css";
 import FrameComponent from "./Frame";
 
 import type { LocalFrame } from "./types";
-import type { Frame } from "../../../types";
 import Badge from "../../shared/Badge";
 
-type FrameLocationProps = { frame: Frame };
+type FrameLocationProps = { frame: LocalFrame };
 function FrameLocation({ frame }: FrameLocationProps) {
-  const library = getLibraryFromUrl(frame);
+  const library = frame.library || getLibraryFromUrl(frame);
```

Comment:
why does this have to change?

Reply:
It needs to change because now figuring out the library is a little more complicated than just looking at the url. The `getLibraryFromUrl` call doesn't actually need to be there at all, because the selector is already doing the work of figuring out the library. I was being defensive by keeping it in with an `||` (perhaps overly so.)